addons/pinniped/config-controller/controllers/controllers.go

```diff
@@ -92,49 +79,50 @@ func (c *PinnipedController) Reconcile(ctx context.Context, req ctrl.Request) (r
 		log.V(1).Info("pinniped-info configmap not found, setting value to nil")
 		pinnipedInfoCM.Data = nil
 	}
-	// if req is empty, CM changed, let's loop through all clusters and create/update/delete secrets
+	// if req is empty, CM changed, let's loop through all secrets and create/update/delete as needed
```

Comment:
How about calling a separate function rather than a comment & all of this inline?